transforms/universal/pii_redactor/python/test/test_pii_redactor_redact_anonamize.py

```diff
@@ -0,0 +1,23 @@
+from test_data import table, redacted_expected_table, expected_metadata_list
+from data_processing.test_support.transform.table_transform_test import (
+    AbstractTableTransformTest,
+)
+from pii_redactor_transform import PIIRedactorTransform, doc_contents_key, redaction_operator_key
+
+
+class TestPIIRedactTransform(AbstractTableTransformTest):
+    """
+    Extends the super-class to define the test data for the tests defined there.
+    The name of this class MUST begin with the word Test so that pytest recognizes it as a test class.
+    """
+
+    def get_test_transform_fixtures(self) -> list[tuple]:
+        fixtures = [
+            (
+                PIIRedactorTransform({doc_contents_key: doc_contents_key, redaction_operator_key: "redact"}),
```

Comment:
can you add another test for the other redactor_operator?

Reply:
@daw3rd It is available here [test_pii_redactor_transform.py](https://github.com/IBM/data-prep-kit/blob/48ddb08ff9d649f9f859b55cd0fffc5d697904dc/transforms/universal/pii_redactor/python/test/test_pii_redactor_transform.py)